template-roles-and-resources.md

```diff
@@ -0,0 +1,167 @@
+
+The current acting templates coordinator is Aaron Peters.
+
+
+### Responsibilities
+
+The templates coordinator:
+- Acts as a project manager for all template-related work.
+- Oversees the overall template development process and provides assistance to contributors working on templates, such as helping get community reviewers or pull request reviewers to look at template drafts.
+- Coordinates and communicates regularly with the template authors, template mentors, community reviewers, and pull request reviewers.
+- Ensures each template receives at least three community reviews on their draft when a template is in the workshop phase.
+- Acts as a gatekeeper for template quality and, along with a template mentor, makes the final judgment call about when a template is ready to move from the workshop phase to the pull request phase.
+- Is authorized to merge template pull requests.
+- Tracks whether the ticket for that template can be closed or if additional tickets need to be created to reflect dependent tasks.
+
+
+### Eligibility
+
+To become the templates coordinator:
+
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should optionally have experience with the template writing process, which is demonstrated by one or more of the following:
+  - Completing a full template set from beginning to end.
+  - Serving as a template mentor for at least 2 months.
+- You must be nominated for this role by the existing templates coordinator, an existing template mentor, or a member of the project steering committee.
+- Receive approval from a majority of members of the project steering committee.
+
+
+## Template mentors
+
+This group of experienced templateers can suggest approaches to challenging template development problems.
+They lead template writing working groups, which often act as workshops where template authors can receive guidance and mentorship while working on templates.
+
+The current template mentors are:
+
+- Aidan Doherty
+- Alyssa Rock
+- Cameron Shorter
+
+
+### Responsibilities
+
+Template mentors:
+- Lead a template writing working group.
+- Mentor other writers and give them the tools, perspectives, and resources to help them create high quality templates.
+- Are very knowledgeable about the template writing policies and procedures and can answer questions that working group members might have about the process.
+- Help connect writers to resources or subject matter experts that can provide feedback on their template drafts.
+- Provide assistance to contributors working on templates in their group, especially when they are blocked.
+- Coordinate and communicate regularly with the templates coordinator.
+- Work with the templates coordinator to ensure each template receives at least three community reviews on their draft when a template is in the workshop phase.

```

Comment:
I would avoid the term "workshop phase" here in favor of "review phase".

Reply:
I think I've changed all instances of workshop phase to feedback phase. Resolving.